Source/WebCore/html/HTMLOptionElement.h

```diff
@@ -64,7 +66,10 @@ class HTMLOptionElement final : public HTMLElement {
 
     String textIndentedToRespectGroupLabel() const;
 
+    // Update 'selectedness'.
```

Comment:
This should be done by selecting an appropriate function name, not with a comment. Like name it `setSelectedness` and then the comment isn’t needed.